aiida_vasp/parsers/tests/test_parser_settings.py

```diff
@@ -4,9 +4,7 @@
 from aiida_vasp.parsers.settings import ParserSettings
 from aiida_vasp.parsers.vasp import DEFAULT_OPTIONS
 
-SETTINGS = {
-    'add_wavecar': True,
-}
+SETTINGS = {'add_wavecar': True, 'critical_errors': {'add_brmix': False}}
```

Comment:
I think we should either try to follow the VASP syntax, or decide what we will call these things. We have and will always have both warnings and errors. In this case VASP generally wants `BRMIX` to be a warning. But this PR is anyway useful as is.

I suggest we go forward and open an issue where we can go through the existing framework to add the granularity needed to act independently on errors and warnings. We can discuss details here: https://github.com/aiida-vasp/aiida-vasp/issues/504